CHANGELOG.md

```diff
@@ -2054,7 +2054,7 @@ In addition to new features, Rocket saw the following smaller improvements:
 
 ## Core
 
-  * Typo in `Outcome` formatting fixed (Succcess -> Success).
```

Comment:
Don't really know if this is false positive or not since it is in CHANGELOG.md and references fixing a typo